keel-clouddriver/src/test/kotlin/com/netflix/spinnaker/keel/clouddriver/ImageServiceTests.kt

```diff
@@ -64,7 +57,7 @@ class ImageServiceTests : JUnit5Minutests {
       tagsByImageId = mapOf(
         "ami-001" to mapOf(
           "build_host" to "https://jenkins/",
-          "appversion" to "my-package-0.0.1~rc.97-h98/JENKINS-job/98",
+          "appversion" to "my-package-0.0.1~rc.97-h98.1c684a9/JENKINS-job/98",
```

Comment:
Took me a while to figure out why this wasn't working correctly. The appversion wouldn't parse

Reply:
Welcome to version scheme hell... 😭